process/project_milestones.md

```diff
@@ -0,0 +1,51 @@
+# Cloud Native Milestones
+
+Milestones are intended to provide checkpoints for projects as they grow and mature in the cloud native ecosystem. When leveraged, projects may find their path towards Graduation is more structured, clear, and allows them to reach maturity in a more robust and well understood manner. Milestones do not guarantee a project’s graduation within the CNCF, rather, they are a collection of key activities that have helped other projects achieve maturity, stability, and adoption. Even after projects are graduated, they may leverage milestones as a mechanism to continue sustaining their project in a manner expected by their community and adopters particularly as they experience change and turnover. The community is welcome to submit PRs to improve and add additional milestones based on their experiences in maturing their projects.
+
+## Sandbox
+
+Some milestones may not apply to all projects and should be leveraged as guiding points. Milestones are described as the desired state with examples for how to achieve this.
+* Project has the basics of security underway
+  * Completion of the [self-assessment](https://github.com/cncf/tag-security/blob/main/assessments/guide/self-assessment.md)
+* Project holds community meetings or active discussions
+  * Zoom with meeting notes occurring more than twice a quarter (or community.cncf.io!) 
+  * Use of GitHub discussions, issues, or a group messaging service (slack, wechat, etc.)
+* Project guides community contributions
+  * PRs from community members receive timely feedback and discussion from the maintainers
+* Project works with an angel adopter
+  * Partners closely with the project to address organizational needs of the project early on
+* Project begins solidifying a versioning schema and release cadence
+  * A roadmap clearly defines initiatives that align with releases
+  * Releases begin to have establish a pattern of regularity
+* Project has engaged a Technical Advisory Group for feedback on some portion of the project relevant to the domain
+* Project has a clearly discoverable governance doc that covers the basics of decision making to include the commit bit
+* TBD - add more!
```

Comment:
```suggestion
* Project has an Adopters file and starts capturing public production users and use cases
* TBD - add more!
```

Reply:
Including this but adjusting language to reflect more project driven implementation with examples of "how"